wrappers/kratos_CFD_wrapper.py

```diff
@@ -615,37 +618,33 @@ def run(self):
         self.fluid_model_part.CloneTimeStep(self.time)
         self.time = self.time + Dt
 
-        for n in xrange(0, substeps):
+        for n in xrange(0, self.CM[CUBA.NUMBER_OF_TIME_STEPS]):
```

Comment:
I like this here....  but it looks like test_kratosWrapper.py (
here https://github.com/simphony/simphony-kratos/pull/9/files#r32045414 ) has not been updated yet